protos-frontend/home/screen_options.proto

```diff
@@ -8,6 +8,8 @@ import "protos-frontend/ui/LocationConsent.proto";
 import "protos-frontend/ui/Button.proto";
 import "protos-frontend/ui/CheckBox.proto";
 import "protos-frontend/ui/Modal.proto";
+import "protos-frontend/ui/Terms.proto";
```

Comment:
This is not need to be imported here for now

Reply:
done